libs/entry-components/date-time-picker/README.md

```diff
@@ -0,0 +1,69 @@
+# Entry Datetime picker
+
+`EntryDateTimePicker` is an Angular component  that provides a custom date time picker implementation
+
+## Setup
+
+Import the [EntryDateTimePicker] in your NgModule:
+
+```typescript
+import { EntryDateTimePickerModule } from "@enigmatry/entry-components/date-time-picker";
+@NgModule({
+    imports: [
+        EntryDateTimePickerModule
+        ,...
+    ],
+    ...
+})
+export class MyModule {}
+```
+
+## Configuration
+
+Provide ENTRY_MAT_DATE_TIME and EntryTimeAdapter in a feature or shared module:
+
+```typescript
+@NgModule({
+  providers: [
+    {
+      provide: ENTRY_MAT_DATE_TIME,
```

Comment:
Injection token names are not correct, should be:
{ provide: EntryTimeAdapter, useClass: EntryNativeTimeAdapter },
{ provide: ENTRY_MAT_DATE_TIME_FORMATS, useValue: dateTimeFormats }